bakery/example/targetservice.go

```diff
@@ -112,7 +112,11 @@ func (srv *targetServiceHandler) writeError(ctx context.Context, w http.Response
 		return
 	}
 
-	herr := httpbakery.NewDischargeRequiredError(m, "/", derr, req)
+	herr := httpbakery.NewDischargeRequiredError(httpbakery.DischargeRequiredErrorParams{
+		Macaroon:      m,
+		OriginalError: derr,
+		Request:       req,
+	})
 	herr.(*httpbakery.Error).Info.CookieNameSuffix = "auth"
```

Comment:
use trhe cookie suffix name in the call above.

Reply:
done